src/base/static/js/views/pages-nav-view.js

```diff
@@ -50,8 +49,4 @@ module.exports = Backbone.View.extend({
     );
   },
 
-  onAuthNavClick: function(evt) {
-    evt.preventDefault();
-    $(".sign-in-menu").toggleClass("is-exposed");
-  },
```

Comment:
This code was not being used - it seems irrelevant to PagesNavView, because it is specific to AuthNavView. Perhaps it was left over from a copy/paste operation?

Either way, it was confusing and unused, so I removed it.